test/cli_inspect_test.go

```diff
@@ -186,3 +186,13 @@ func (suite *PouchInspectSuite) TestContainerInspectState(c *check.C) {
 	output = command.PouchRun("inspect", "-f", "{{.State.Status}}", name).Stdout()
 	c.Assert(strings.TrimSpace(output), check.Equals, "stopped")
 }
+
+func (suite *PouchInspectSuite) TestContainerInspectPorts(c *check.C) {
+	name := "TestContainerInspectPorts"
+	res := command.PouchRun("run", "-d", "--name", name, "-p", "8080:80", busyboxImage, "top")
```

Comment:
Since `res` not used by other `code`, how about `command.PouchRun("run", "-d", "--name", name, "-p", "8080:80", busyboxImage, "top").Assert(c, icmd.Success)` ?

Reply:
> Since `res` not used by other `code`, how about `command.PouchRun("run", "-d", "--name", name, "-p", "8080:80", busyboxImage, "top").Assert(c, icmd.Success)` ?

Reasonable, I will fix it.